geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/concurrent/ConcurrentParallelGatewaySenderOperation_1_DUnitTest.java

```diff
@@ -649,7 +655,7 @@ public void testParallelPropagationSenderStartAfterStopOnAccessorNode() throws T
     vm7.invoke(() -> WANTestBase.stopSender("ln"));
 
     // SECOND RUN: do some of the puts after the senders are stopped
-    vm4.invoke(() -> WANTestBase.doPuts(getTestMethodName() + "_PR", 1000));
+    vm4.invoke(() -> WANTestBase.doPuts(getTestMethodName() + "_PR", 800));
```

Comment:
just curious why this needed to be changed to 800?

Reply:
There is no need to change this. I will return old value.